scripts/variantstore/AOU_GVS_WORKSPACE.md

```diff
@@ -101,30 +130,34 @@ This is done by running the `GvsCreateFilterSet` workflow with the following par
 
 **Note:** This workflow does not use the Terra Entity model to run, so be sure to select `Run workflow with inputs defined by file paths`
 
+Sometimes this workflow will fail because the Gaussians have not cenverged. Dont panic! It can happen to anyone's data!
+The first step in this case will be to adjust the Guassian for the failed step (there are two possible steps: model creation for the SNPS and model creation for the InDels) to a lower number.
```

Comment:
Please include the specific inputs names for the gaussian values to change so that the user can fill them out easily.